### DISCUSSION TITLE
VASP Application Processing Time

### DISCUSSION THREAD

**Discussion by NotableVentures** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#42] [General]
VASP application processing times tremendously impact the attractiveness of a jurisdiction.  CIMA's long processing times -- I understand some applications have been unprocessed for nearly a year -- have already discouraged many businesses from setting up in the Cayman Islands. Any new VASP rule update is much less effective if the application process doesn't have service levels for processing times.

**Comment by Clearwaterscrypto** [comment]
Speed to market is a critical factor for service providers when selecting the best jurisdictions from where to operate. CIMA should aim to determine the outcome of applications within 3-6months if Cayman is to be an attractive destination and remain competitive with Bermuda and the Bahamas.
